app/views/passengers/_form.html.haml

```diff
@@ -30,13 +31,15 @@
     = form.label :status
     = form.select :status, Passenger::STATUSES, { include_blank: true }, { id: :passenger_status }
   .field
-    = form.label :registered_with_disability_services
-    = form.check_box :registered_with_disability_services,
-      id: :pax_registered_with_DS, disabled: @passenger.permanent?
+    - unless @passenger.permanent?
+      = form.label :registered_with_disability_services
+      = form.check_box :registered_with_disability_services,
+        id: :pax_registered_with_DS, disabled: @passenger.permanent?
   .field
-    = form.label :has_brochure
-    = form.check_box :has_brochure,
-      id: :pax_has_brochure, disabled: @passenger.permanent?
+    - unless @passenger.permanent?
+      = form.label :has_brochure
+      = form.check_box :has_brochure,
+        id: :pax_has_brochure, disabled: @passenger.permanent?
```

Comment:
I'm sensing a pattern here... am I wrong about this?

Reply:
Nope. It's up to you and @chauce if I should hide these or just keep it the way it was (disabling boxes).

Reply:
My personal preference is that the box is just disabled but I feel not strongly at all about this. I can just imagine maybe _maybe_ some confusion on the part of the dispatcher at some point "hey, don't I have to check if this person is registered or has a brochure?" but it's really whatever.

Reply:
Ok. That second part is what I was thinking about, too.